sanic/deprecated.py

```diff
@@ -56,10 +58,11 @@ def from_pyfile(self, filename: str) -> True:
     try:
         with open(filename) as config_file:
             exec(  # nosec
-                compile(config_file.read(), filename, "exec"), module.__dict__,
+                compile(config_file.read(), filename, "exec"),
+                module.__dict__,
             )
     except IOError as e:
-        e.strerror = f"Unable to load configuration file (e.strerror)"
+        e.strerror = "Unable to load configuration file (e.strerror)"
```

Comment:
Shouldn't it be ?:

            e.strerror = "Unable to load configuration file {e.strerror}"

Reply:
@ahopkins @myusko 
Do not know if it is to late, but this is not "bleeding bug", but please also take a look.